### PR TITLE
fixes #262. Make Ownable's owner configurable in construction time

### LINKED ISSUE #262: Make Ownable's owner configurable in construction time
Right now, `msg.sender` is made the contract's owner. Make `Ownable`'s constructor receive an address

### PR DESCRIPTION
**Pull request by rstormsf**
fixes #262

### CODE REVIEW

**Comment by rstormsf** [comment]
:-( tests failing. 

```
r: The contract code couldn't be stored, please check your gas amount.
    at Object.callback (/home/travis/build/OpenZeppelin/zeppelin-solidity/node_modules/web3/lib/web3/contract.js:147:46)
    at /home/travis/build/OpenZeppelin/zeppelin-solidity/node_modules/web3/lib/web3/method.js:142:25
    at /home/travis/build/OpenZeppelin/zeppelin-solidity/node_modules/web3/lib/web3/requestmanager.js:89:9
    at XMLHttpRequest.request.onreadystatechange (/home/travis/build/OpenZeppelin/zeppelin-solidity/node_modules/web3/lib/web3/httpprovider.js:118:13)
    at XMLHttpRequestEventTarget.dispatchEvent (/home/travis/build/OpenZeppelin/zeppelin-solidity/node_modules/xhr2/lib/xhr2.js:64:18)
    at XMLHttpRequest._setReadyState (/home/travis/build/OpenZeppelin/zeppelin-solidity/node_modules/xhr2/lib/xhr2.js:354:12)
    at XMLHttpRequest._onHttpResponseEnd (/home/travis/build/OpenZeppelin/zeppelin-solidity/node_modules/xhr2/lib/xhr2.js:509:12)
    at IncomingMessage.<anonymous> (/home/travis/build/OpenZeppelin/zeppelin-solidity/node_modules/xhr2/lib/xhr2.js:469:24)
    at emitNone (events.js:91:20)
    at IncomingMessage.emit (events.js:185:7)

```

**Comment by maraoz** [comment]
Did you run the tests locally? If so, make sure to have the correct version of truffle to reproduce the test error locally

File: contracts/ownership/Ownable.sol [hunk 1]
```diff
@@ -14,8 +14,12 @@ contract Ownable {
    * @dev The Ownable constructor sets the original `owner` of the contract to the sender
    * account.
    */
-  function Ownable() {
-    owner = msg.sender;
+  function Ownable(address _owner) {
+    if(_owner != 0){
+      owner = _owner;
```
Comment (DavidKnott) [inline contracts/ownership/Ownable.sol]: I believe you're gonna want to compare the address with 0x0

Comment (rstormsf) [inline contracts/ownership/Ownable.sol]: Ohh that's right! thank you @DavidKnott

**Comment by rstormsf** [comment]
#262 - still not sure which direction we are going, since this PR is invalid and based on prev commit, I am closing it.

**Pull request closed without merging**